cluster/photon-controller/config-default.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##########################################################
+#
+# Common parameters for Kubernetes
+#
+##########################################################
+
+# Default number of minion nodes to make. You can change this as needed
+NUM_NODES=2
+
+# Range of IPs assigned to pods
+NODE_IP_RANGES="10.244.0.0/16"
+
+# IPs used by Kubernetes master
+MASTER_IP_RANGE="${MASTER_IP_RANGE:-10.246.0.0/24}"
+
+# Range of IPs assigned by Kubernetes to services
+SERVICE_CLUSTER_IP_RANGE="10.244.240.0/20"
+
+##########################################################
+#
+# Advanced parameters for Kubernetes
+#
+##########################################################
+
+INSTANCE_PREFIX=kubernetes
+
+# Master VM parameters
+MASTER_MEMORY_MB=1024
```

Comment:
Since this setup is only going to have one Kubernetes master, it will be better to use a larger VM to host the master. 1GB memory and 1 Core seems too less. The API-Server especially will need more processing power, since the minions continuously maintain a heart beat with the API-Server.